worldeditadditions_commands/commands/selectors/sfactor.lua

```diff
@@ -0,0 +1,105 @@
+-- ███████ ███████  █████   ██████ ████████  ██████  ██████
+-- ██      ██      ██   ██ ██         ██    ██    ██ ██   ██
+-- ███████ █████   ███████ ██         ██    ██    ██ ██████
+--      ██ ██      ██   ██ ██         ██    ██    ██ ██   ██
+-- ███████ ██      ██   ██  ██████    ██     ██████  ██   ██
+local wea = worldeditadditions
+worldedit.register_command("sfactor", {
+	params = "<mode> <factor> [<target=xz>]",
+	description = "Make the length of one or more target axes of the current selection to be multiple(s) of <factor>.",
+	privs = { worldedit = true },
+	require_pos = 2,
+	parse = function(params_text)
+		local parts = wea.split(params_text, "%s+", false)
+		
+		--- Dissable command for current release
+		if false then
+			return false, "Feature not yet implemented."
+		end
```

Comment:
I've done the release, so it's fine if  it doesn't work properly just yet - just as long as Minetest doesn't crash on startup and any bugs are unlikely to affect (too many) other commands :-)

Reply:
I'll get rid of that... I should probably take the tests out of smake too.